modules/notifications/module.php

```diff
@@ -0,0 +1,86 @@
+<?php
+namespace Elementor\Modules\Notifications;
+
+use Elementor\Core\Base\Module as BaseModule;
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly.
+}
+
+class Module extends BaseModule {
+
+	public function get_name() {
+		return 'notification-center';
+	}
+
+	public function __construct() {
+		parent::__construct();
+
```

Comment:
```suggestion

		/**
		 * Before admin top bar enqueue scripts.
		 *
		 * Fires before Elementor admin top bar  scripts are enqueued.
		 *
		 * @since 3.19.0
		 */
```